content/project-organization/quality-council/processes/issue-management.md

```diff
@@ -120,126 +135,132 @@ The handling of issues follows a detailed process. Here are the different steps,
 
-#### Connecting an issue to an EPIC
+### Connecting an issue to an EPIC
 
-When the issue is a **bug that was reproduced** (meaning with the `Verified` label), it MUST in most cases be connected to an `EPIC` if it already exists.
+{{% notice warning %}}
+When the issue is a **bug that was reproduced** (meaning with the `Verified` label), it MUST in most cases be connected to an Epic if it already exists.
+{{% /notice %}}
 
-To do this, perform a search using this filter [EPIC list](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aopen+is%3Aissue+label%3AEPIC) where all `EPIC` are listed. If there is a related `EPIC`, add the issue to the `EPIC` (by editing the description) and ping the product team.
+To do this, perform a search in the [Bug classification board](https://github.com/PrestaShop/PrestaShop/projects/11) where all EPIC are listed. If there is a related EPIC, add the issue to the EPIC (by editing the description) and ping the product team.
 
-If no `EPIC` clearly corresponds to the issue (or in case of doubt), just add the bug issue to the 1st column of the [kanban](https://github.com/PrestaShop/PrestaShop/projects/6) called "PrestaShop Next Major". The product team will regularly review the issues in this kanban and create new `EPIC`s when needed.
+If no Epic clearly corresponds to the issue (or in case of doubt), just add the bug issue to the 1st column of the kanban called "waiting for classification". The product team will regularly review the issues in this kanban and create new EPICs when needed.
 
-Note that the product team must be able to understand the bug to be able to classify it in an `EPIC`. This means that if the issue is not correctly described nor labelled, or if this is a very tech issue, it **MUST NOT be added** to the kanban.
+Note that the product team must be able to understand the bug to be able to classify it in an EPIC. This means that if the issue is not correctly described nor labelled, or if this is a very tech issue, it MUST NOT be added to the kanban.
 
-#### Rewriting the issue
+### Rewriting the issue
 
-Once all the steps have been completed, the content of the issue should be rewritten if it is unclear, or if the author has clarified in the comments. All necessary information should be available in the body of the issue! It may be necessary **to rewrite the title if it is too generic** (e.g. "Help! BUG!") or if it does not accurately reflect the bug. Pay special attention to the **reproduction steps** ("how to reproduce")!
+Once all the steps have been completed, the content of the issue should be rewritten if it is unclear, or if the author has clarified in the comments. All necessary information should be available in the body of the issue!It may be necessary to rewrite the title if it is too generic (e.g. "Help! BUG!") or if it does not accurately reflect the bug.Pay special attention to the **reproduction steps** ("how to reproduce")!
```

Comment:
`issue! It may be necessary`

missing space

Reply:
Thanks